.travis.yml

```diff
@@ -24,4 +24,4 @@ notifications:
     on_success: never
     on_failure: always
 after_success:
-- ". ./fastlane/travis scripts/update-test-app.sh"
+- ". ./fastlane/travis-scripts/update-test-app.sh"
```

Comment:
What we are going to do with the test app?

Reply:
Probably nothing for now.